dotCMS/src/main/java/com/dotcms/workflow/form/WorkflowReorderWorkflowActionStepForm.java

```diff
@@ -0,0 +1,42 @@
+package com.dotcms.workflow.form;
+
+import com.dotcms.repackage.com.fasterxml.jackson.annotation.JsonProperty;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+
+@JsonDeserialize(builder = WorkflowReorderWorkflowActionStepForm.Builder.class)
+public class WorkflowReorderWorkflowActionStepForm extends Validated {
+
+
+    @NotNull
+    private final int           order;
+
+    public int getOrder() {
+        return order;
+    }
+
+
+    public WorkflowReorderWorkflowActionStepForm(final Builder builder) {
+
+        this.order              = builder.order;
+        this.checkValid();
+    }
+
+    public static final class Builder {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)